src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -74,6 +82,28 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
         }, 500);
     }
 
+    loadFirstMessage(): void {
+        const firstMessageContent = {
+            textContent: 'artemisApp.exerciseChatbot.firstMessage',
+            type: IrisMessageContentType.TEXT,
+        } as IrisMessageContent;
+
+        const firstMessage = {
+            sender: IrisSender.LLM,
+            id: 0,
+            content: [firstMessageContent],
+            sentAt: dayjs(),
+        } as IrisServerMessage;
+
+        if (this.messages.length === 0) {
+            this.isFirstMessage = true;
+            this.stateStore.dispatch(new ActiveConversationMessageLoadedAction(firstMessage));
+        } else if (this.messages[0].id === 0) {
```

Comment:
In general it is not a good practice to hardcode some id values. Is there a better way to ensure that this is the message you need?